fw/http_parser.h

```diff
@@ -26,6 +26,7 @@
 typedef struct {
 	unsigned int	size;	/* number of elements in the table */
 	unsigned int	off;
+	unsigned int	cnt;
```

Comment:
Probably we'll have 4 bytes hole here, so I'm wondering if we do support 2G headers and can't we just use `unsigned short` at least for `cnt` and `off`?

Reply:
No problem provided we do not need bigger number of headers

Fixed